authenticator/components/application-list-icon.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { StyleSheet, Image, View } from 'react-native';
+
+export default function ApplicationListIcon( id ) {
+    return(
+        <View>
+            <Image style={styles.logo} source={{uri: 'https://reactnative.dev/img/tiny_logo.png'}}/>
+        </View>
+    );
+}
+
+const styles = StyleSheet.create({
+    logo: {
+      width: 75,
+      height: 75,
+    },
+  });
```

Comment:
this is simple enough to add directly to the application list item component

Reply:
I can see the advantages of abstracting this to a component though, especially if we're using Icons throughout the website, so it definitely could work to our advantage in the long run. Let me know what you think about this!

Reply:
The main reason I separated it here was I assumed that we would need a way to get an icon.  My idea was to somehow use the favicon from the site (similar to the way Bitwarden password manager gets icons).  I thought the logic for that might be better abstracted away in this file eventually.

Reply:
Ah ok that makes a lot of sense, let's go with that